index.bs

```diff
@@ -2037,24 +2058,37 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |options| be the value of <code>|options|.{{CredentialRequestOptions/publicKey}}</code>.
 
-1. If the {{PublicKeyCredentialRequestOptions/timeout}} member of |options| is present, check if its value lies
-    within a reasonable range as defined by the [=client=] and if not, correct it to the closest value lying within that range.
-    Set a timer |lifetimeTimer| to this adjusted value. If the {{PublicKeyCredentialRequestOptions/timeout}} member of
-    |options| is not present, then set |lifetimeTimer| to a [=client=]-specific default.
+1. If <code>|options|.{{CredentialRequestOptions/mediation}}</code> is present with the value
+    {{CredentialMediationRequirement/conditional}}:
 
-        Recommended ranges and defaults for the {{PublicKeyCredentialRequestOptions/timeout}} member of |options| are as follows.
-          If <code>|options|.{{PublicKeyCredentialRequestOptions/userVerification}}</code>
-            <dl class="switch">
-                :   is set to {{UserVerificationRequirement/discouraged}}
-                ::  Recommended range: 30000 milliseconds to 180000 milliseconds.
-                ::  Recommended default value: 120000 milliseconds (2 minutes).
+    1. If <code>|options|.{{PublicKeyCredentialRequestOptions/allowCredentials}}</code> is not [=list/empty=],
+        return a {{DOMException}} whose name is "{{NotSupportedError}}", and terminate this algorithm.
```

Comment:
Ah, I see. Indeed stateless credentials would be hard to integrate in an account picker flow. Supporting a non-empty `allowCredentials` would enable CTAP2 creds with `credProtect` level 2 to work with this flow, though I guess that's a bit niche since the main use case for this is to use discoverable credentials without first identifying the user. And you're definitely right that it's easier to add the feature later than to remove it once introduced.